src/store/elementalChat.js

```diff
@@ -192,47 +157,21 @@ export default {
         })
         .catch(error => log('createChannel zome error', error))
     },
-    getMessageChunk: async ({ state, commit, rootState, dispatch }, { channel, latestChunk, activeChatter, firstChunkLoad }) => {
-      if (!latestChunk) {
-        latestChunk = 0
-      }
-
-      const channelMsgCount = channel.currentMessageCount || CHUNK_SIZE
-      const chunkRemainder = calculateRemainder(channelMsgCount)
-      const chunkQuotient = calculateQuotient(channelMsgCount)
-      const loadedChunkEarliest = chunkRemainder ? chunkQuotient + 1 : chunkQuotient
-      const loadedChunkOldest = loadedChunkEarliest - 1
-
-      let chunkStart = (firstChunkLoad && latestChunk > 0)
-        ? latestChunk - 1
-        : firstChunkLoad
-          ? latestChunk
-          : (loadedChunkEarliest === 1)
-            ? 0
-            : latestChunk - loadedChunkEarliest
-
-      let chunkEnd = (firstChunkLoad || latestChunk === 0)
-        ? latestChunk
-        : chunkStart === 0
-          ? 1
-          : latestChunk - loadedChunkOldest
-
-      if (chunkStart < 0) {
-        chunkStart = 0
-        chunkEnd = 1
+    listMessages: async ({ state, commit, rootState, dispatch }, { channel, earliest_seen, target_message_count }) => {            
+      const payload = { 
+        channel: channel.entry,
+        earliest_seen,
+        target_message_count: target_message_count || 20,
       }
 
-      if (chunkStart >= 0) {
-        const payload = { channel: channel.entry, chunk: { start: chunkStart, end: chunkEnd }, active_chatter: activeChatter || true }
-        return callZome(dispatch, rootState, 'chat', 'list_messages', payload, 50000)
-          .then(async result => {
-            if (result) {
-              // NOTE: messages will be aggregated with current messages in following chain of fns
-              handleListMessagesResult(state, commit, channel.entry.uuid, result.messages)
-            }
-          })
-          .catch(error => log('listMessages zome error', error))
-      }
+      return callZome(dispatch, rootState, 'chat', 'list_messages', payload, 50000)
+        .then(async result => {
```

Comment:
Does not need to be async
```suggestion
        .then(result => {
```